src/main/java/org/mule/tools/apikit/output/MuleConfigGenerator.java

```diff
@@ -68,113 +54,62 @@
                                                                                                        "http://www.mulesoft.org/schema/mule/ee/core"),
                                                                                      "http://www.mulesoft.org/schema/mule/ee/core/current/mule-ee.xsd");
 
-  private static final String INDENTATION = "    ";
-
   private final List<GenerationModel> flowEntries;
-  private final File rootDirectory;
-  private final Set<File> ramlsWithExtensionEnabled;
   private final RuntimeEdition runtimeEdition;
-  private final List<API> apis;
-  private final List<ParsingIssue> errors = new LinkedList<>();
+  private final List<ApikitMainFlowContainer> apis;
+  private List<MuleConfig> muleConfigsInApp = new ArrayList<>();
+
+  private final List<ScaffoldingError> scaffoldingErrors = new ArrayList<>();
```

Comment:
always empty list

Reply:
Yes, most of the scaffolding errors of the _old_ scaffolder were due to problems parsing the ramls. Now, all the files parsing logic is in the ParserService, so those problems are gone. 

I decided to leave that list (which is currently empty) because there may be errors scaffolding the munit tests.